tests/framework/web/UrlRuleTest.php

```diff
@@ -513,6 +513,20 @@ protected function getTestsForCreateUrl()
                 ],
             ],
             [
+                'single optional param with suffix',
+                [
+                    'pattern' => '<page>',
+                    'route' => 'page/view',
+                    'defaults' => ['page' => 'index'],
+                    'suffix' => '/',
+                ],
+                [
+                    ['page/view', [], '/'],
```

Comment:
This assertion is incorrect. Only parameters with empty default values may be omitted.